src/components/ErrorBoundary/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { Component, ErrorInfo, ReactNode } from 'react';
+import ErrorIcon from '../icons/ErrorBoundaryIcon';
+import styles from './ErrorBoundary.module.scss';
+
+const { errorBoundary, message, content } = styles;
+
+type Props = {
+  children?: ReactNode;
+};
+type State = {
+  hasError: boolean;
+};
```

Comment:
I suggest creating a separate file within this folder and moving the types into this file and maybe giving the types a bit narrower names, e.g. `ErrorBoundaryProps`, `ErrorBoundaryState`